packages/e2e-tests/tests/chat-list-multiselect.spec.ts

```diff
@@ -30,10 +31,14 @@ const expectSelectedChats = async (chatNums: number[]) => {
   )
 }
 
-test.beforeAll(async ({ browser }) => {
+test.beforeAll(async ({ browser, chatmail }) => {
   const contextForProfileCreation = await browser.newContext()
   const pageForProfileCreation = await contextForProfileCreation.newPage()
 
+  console.log(
+    `Running multiselect tests with ${chatmail ? 'chatmail' : 'plain email'} profiles`
+  )
+
```

Comment:
Are these logs useful? Why is it added in some files but not others?

Reply:
I added them for developement and reviews.  Can be removed